patient-appointment-management/assets/index.html

```diff
@@ -181,6 +204,160 @@ <h3>Troubleshooting</h3>
     </section>
 
   </main>
+  <simulate>
```

Comment:
`<simulate>` is not an HTML element. Are you intending for this to be `<div class="simulate">` instead?

Reply:
changed to an id.

Reply:
not sure, I'll ask the developer who wrote this. can you add 'jaharap' to access this repo? https://issues.corp.twilio.com/browse/HD-27
thx